ecl2df/common.py

```diff
@@ -621,3 +622,46 @@ def stack_on_colnames(dframe, sep="@", stackcolname="DATE", inplace=True):
     del dframe["level_0"]
     dframe.index.name = ""
     return dframe
+
+
+def parse_zonemapfile(filename: str):
```

Comment:
```suggestion
def parse_zonemapfile(filename: str) -> Dict[int, str]:
```
...if this is what is returned?

Reply:
Will await this, until the whole of ecl2df is type hinted.

Reply:
https://github.com/equinor/ecl2df/issues/290